frontend/src/AddMovie/AddNewMovie/AddNewMovieSearchResult.js

```diff
@@ -65,14 +67,21 @@ class AddNewMovieSearchResult extends Component {
       images,
       isExistingMovie,
       isExclusionMovie,
-      isSmallScreen
+      isSmallScreen,
+      colorImpairedMode,
+      queueItems,
+      id,
+      monitored,
+      hasFile,
+      isAvailable
     } = this.props;
 
     const {
       isNewAddMovieModalOpen
     } = this.state;
 
     const linkProps = isExistingMovie ? { to: `/movie/${titleSlug}` } : { onPress: this.onPress };
+    const queueItem = queueItems.find((q) => q.movieId === id);
```

Comment:
This can be done in AddNewMovieSearchResultConnector once you move the selector there